{"name": "test_login_non_existing_user", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Error: The u...ress instead.' == 'Ereror: The ...ress instead.'\n  - Ereror: The username TestUser is not registered on this site. If you are unsure of your username, try your email address instead.\n  ?   -\n  + Error: The username TestUser is not registered on this site. If you are unsure of your username, try your email address instead.", "trace": "self = <test_login_negative.TestLoginNegative object at 0x10a09e190>\n\n    @pytest.mark.tcid12\n    def test_login_non_existing_user(self):\n        print(\"************\")\n        print(\"TEST LOGIN NON EXISTING\")\n        print(\"************\")\n        my_account = MyAccountSignedOut(self.driver)\n        my_account.go_to_my_account()\n        my_account.input_login_username(\"TestUser\")\n        my_account.input_login_password(\"TestPassword1234*\")\n        my_account.click_login_button()\n        err_text = my_account.get_err_text()\n        expected_err_text = 'Ereror: The username TestUser is not registered on this site. If you are unsure of your ' \\\n                              'username, try your email address instead.'\n>       assert err_text == expected_err_text\nE       AssertionError: assert 'Error: The u...ress instead.' == 'Ereror: The ...ress instead.'\nE         - Ereror: The username TestUser is not registered on this site. If you are unsure of your username, try your email address instead.\nE         ?   -\nE         + Error: The username TestUser is not registered on this site. If you are unsure of your username, try your email address instead.\n\ntests/my_account/test_login_negative.py:20: AssertionError"}, "attachments": [{"name": "screenshot", "source": "21fb27f4-6fd3-415c-a721-0583a459db0a-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "0f2e7c8e-e150-45be-a12d-fc2e8d3edd50-attachment.txt", "type": "text/plain"}], "start": 1688028017558, "stop": 1688028020215, "uuid": "128d64a4-1152-4a88-8dd8-94fbcbf11633", "historyId": "5dc27d4950019336d5097f28663ae4b9", "testCaseId": "5dc27d4950019336d5097f28663ae4b9", "fullName": "tests.my_account.test_login_negative.TestLoginNegative#test_login_non_existing_user", "labels": [{"name": "tag", "value": "tcid12"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "tests.my_account"}, {"name": "suite", "value": "test_login_negative"}, {"name": "subSuite", "value": "TestLoginNegative"}, {"name": "host", "value": "Roamer.local"}, {"name": "thread", "value": "30740-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.my_account.test_login_negative"}]}